llvm/lib/Target/RISCV/RISCVInstrInfoXCheri.td

```diff
@@ -1554,3 +1554,21 @@ let Predicates = [HasCheri, IsRV64, IsCapMode] in {
 defm : CheriLdPat<load, CLC_128>;
 defm : CheriStPat<store, CSC_128, GPCR>;
 } // Predicates = [HasCheri, IsRV64, IsCapMode]
+
+
+/// CHERI memory colouring extensions
+let Predicates = [HasCheri, IsRV64] in {
+def CGetVersion   : Cheri_r<0x13, "cgetversion">;
+def CSetVersion   : Cheri_rr<0x2, "csetversion">;
+let mayLoad = 1 in
+{
+def CLoadVersion  : Cheri_r<0x14, "cloadversion">;
+def CLoadVersions : Cheri_r<0x15, "cloadversions">;
+}
+let mayStore = 1, mayLoad = 0, hasSideEffects = 0 in
+def CStoreVersion : RVInstCheriTwoSrc<0x7e, 0x2, 0, OPC_CHERI, (outs),
+                                (ins GPR:$rs1, GPCR:$rs2),
```

Comment:
Indentation

Reply:
Is the canonical indentation to aligh with the `<` ?

Reply:
If you mean lining the first `0` up with the `(` then yes

Reply:
Fixed.